core/edit/edit_paragraph.py

```diff
@@ -0,0 +1,108 @@
+from talon import Context, Module, actions
+
+ctx = Context()
+mod = Module()
+
+
+@ctx.action_class("edit")
+class EditActions:
+    def paragraph_start():
+        if extend_paragraph_start():
+            actions.edit.left()
+
+    def paragraph_end():
+        if extend_paragraph_end():
+            actions.edit.right()
+
+    def select_paragraph():
+        if is_line_empty():
+            return
+        # Search for start of paragraph
+        actions.edit.extend_paragraph_start()
+        actions.edit.left()
+        # Extend to end of paragraph
+        actions.edit.extend_paragraph_end()
+
+    def extend_paragraph_start():
+        extend_paragraph_start()
```

Comment:
You might want to explain why you don't use the default Talon supplied action.

Reply:
done